tracer/src/Datadog.Trace/IAST/Aspects/System/StringAspects.cs

```diff
@@ -254,36 +255,7 @@ public static string Concat(object param1, object param2, object param3, object
     {
         return StringModuleImpl.OnStringConcat(new StringConcatParams(param1?.ToString(), param2?.ToString(), param3?.ToString(), param4?.ToString()), string.Concat(param1, param2, param3, param4));
     }
-
```

Comment:
For all the frameworks, when we call concat with 5 parameters, the overload with object[] is called

Reply:
🤯 maybe you could add this as a comment in the summary or so?

Reply:
Added!